dotnet/SimpleSauce.Test/SauceSessionTests.cs

```diff
@@ -9,6 +11,9 @@ namespace SimpleSauce.Test
     public class SauceSessionTests : BaseTest
     {
         private Mock<ISauceRemoteDriver> _dummyDriver;
+        private string _browserOptionsSetInSauceJson;
+        private Root _browserOptionsSetInSauce;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the field '_browserOptionsSetInSauce' and declare it as a local variable in the relevant methods.](https://app.codacy.com/app/SauceLabs/simple_sauce/pullRequest?prid=4626672)